### ISSUE TITLE
mAP 83.4

### ISSUE THREAD

**Issue by Annieliaquat** [tensorflow/models#10775]
I am using [SSD MobileNet V2 FPNLite 320x320] to train my model. I have chest x-ray to detect Covid-19. There 1349 Normal chest x-rays and 3883 Covid-19 chest x-rays. I have used different Augmentations to increase my Normal chest-xray from 1349 to 2215. and pneumonia images from 3883 to 4032. 
I have trained my model 4000 training steps, and it has training loss of 0.21 and evaluation loss of 0.23. 

I am confused that I have gotten 0.837 mAP. Is it a good result? I need to compare my results with some other papers, but they have accuracy nearest 96 or 98. There result is in accuracy and mine is in mAP. How am I going to compare them??

![image](https://user-images.githubusercontent.com/43025113/189542174-62298b2e-f953-4a5d-8b0e-7c84986d9269.png)

Another question is that most of the Tensorflow object detection API has mAP between 0.20 to 0.50, but mine is 0.83. So is their some issue with my model or its fine? Because it is detecting and classifying most of the images accurately. 

Please please any expert guide me regarding all of the issues I have asked.

**Comment by sushreebarsa** [comment]
@Annieliaquat 
In order to expedite the trouble-shooting process, could you please provide the entire URL of the repository which you are using. Please provide more details on the issue reported here and share the error log in text instead of screenshot which will help us to analyse the issue? Thanks!
